doc/source/tutorial/stats/continuous_jf_skew_t.rst

```diff
@@ -0,0 +1,35 @@
+
+.. _continuous-jf_skew_t:
+
+Jones and Faddy Skew-T Distribution
+===================================
+
+A skew extension of the `t` distribution, defined for :math:`a>0` and
+:math:`b>0`.
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
+    f(x;a,b) & = & C_{a,b}^{-1} \left(1+\frac{x}{\left(a+b+x^2\right)^{1/2}}\right)^{a+1/2} \left(1-\frac{x}{\left(a+b+x^2\right)^{1/2}}\right)^{b+1/2} \\
+    F(x;a,b) & = & I\left(\frac{1+x(a+b+x^2)^{-1/2}}{2};a,b\right) \\
```

Comment:
Looks like it matches:
<img width="538" alt="image" src="https://github.com/scipy/scipy/assets/6570539/4c4dc6e3-3840-4adb-a2dd-c273f985f3dc">